nmdc_runtime/util.py

```diff
@@ -510,7 +510,27 @@ def ensure_unique_id_indexes(mdb: MongoDatabase):
             collection_name in schema_collection_names_with_id_field()
             or all_docs_have_unique_id(mdb[collection_name])
         ):
-            mdb[collection_name].create_index("id", unique=True)
+            # Check if index already exists, and if so, drop it if not unique
+            try:
+                existing_indexes = list(mdb[collection_name].list_indexes())
+                id_index = next(
+                    (idx for idx in existing_indexes if idx["name"] == "id_1"), None
+                )
+
+                if id_index:
```

Comment:
I noticed sometimes my `make up-test` & `make test` commands might fail if indexes were already in place in my database.  This is just a bit of safety coding; happy to remove.